src/Evolution/Systems/GrMhd/ValenciaDivClean/KastaunEtAl.cpp

```diff
@@ -263,7 +276,9 @@ Primitives FunctionOfMu<ThermodynamicDim>::primitives(const double mu) const {
     p_hat = get(equation_of_state_.pressure_from_density_and_energy(
         Scalar<double>(rho_hat), Scalar<double>(epsilon_hat)));
   } else if constexpr (ThermodynamicDim == 3) {
-    ERROR("3d EOS not implemented");
+    p_hat = get(equation_of_state_.pressure_from_density_and_energy(
+        Scalar<double>(rho_hat), Scalar<double>(epsilon_hat),
+        Scalar<double>(electron_fraction_)));
```

Comment:
You can ignore the "c-style casts are discouraged" warnings from clang-tidy (we'll have to fix those for tensors like this).

Reply:
Do you mean literally ignore them or NOLINT them?  :)

Reply:
Ignore, no need to nolint because we have to fix those more permanently (this patter occurs all over the code, can't nolint it everywhere)